portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACIndividualsDashboard.path

```diff
@@ -39,6 +39,21 @@
     <td>//div[@class='card card-root distribution-card-root']//li/button[text()='${key_breakdownTabName}']</td>
     <td></td>
 </tr>
+<tr>
+    <td>INDIVIDUALS_TAB</td>
+    <td>//ul[@class='nav-root page-subnav nav nav-underline']//li[contains(@class,'nav-item') and contains(.,'${key_tabName}')]</td>
+    <td></td>
+</tr>
+<tr>
+    <td>KNOWN_INDIVIDUALS_LIST_ITEMS</td>
+    <td>//div[contains(@class,'table-title text-truncate') and contains(.,'${key_individualName}')]</td>
```

Comment:
hey @calvinctvo , when using contains, try to use only one `class` (e.g.: `'table-title'` or `'text-truncate'`), so your selector will less fragile or susceptible to break if one of the classes change.

The same applies to the `INDIVIDUALS_TAB` selector.